client/onboarding/index.tsx

```diff
@@ -37,15 +36,21 @@ const OnboardingStepper = () => {
 		persistFlowState( step, data );
 	};
 
+	const initialStep = () => {
```

Comment:
Good thinking to handle this! We can probably remove it in a future version, once enough time has passed that people won't be likely to have `mode` saved as this value.

Reply:
I'll add comment about it!

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/8378/commits/6f1652e0822ab5b3f229d1d2e1dee2e23f57ecd0